src/DonationForms/Blocks/DonationFormBlock/resources/editor/Edit.tsx

```diff
@@ -93,7 +94,16 @@ export default function Edit({attributes, isSelected, setAttributes, className,
                     )}
                 </>
             ) : (
-                <DonationFormSelector formOptions={formOptions} isResolving={isResolving} handleSelect={handleSelect} />
+                <EntitySelector
+                    id={'formId'}
+                    label={__('Choose a donation form', 'give')}
+                    options={formOptions}
+                    isLoading={isResolving}
+                    emptyMessage={__('No donation forms were found.', 'give')}
+                    loadingMessage={__('Loading Donation Forms...', 'give')}
+                    buttonText={__('Confirm', 'give')}
+                    onConfirm={handleSelect}
+                />
```

Comment:
Let's add an `unreleased` tag in the top of this export to document this change.

Reply:
Resolved: d21f3a1c5f3014c533e26166246326171a95895c